R/tt_dotabulation.R

```diff
@@ -1585,7 +1598,7 @@ splitvec_to_coltree <- function(df, splvec, pos = NULL,
     vals <- rawpart[["values"]]
     labs <- rawpart[["labels"]]
 
-
+    force(alt_counts_df)
```

Comment:
why? to avoid lazy evaluation?

Reply:
I think its because we recursively call it inside that mapply. I'm not entirely sure that its needed there, tbh, but even if it isn't it isn't hurting anything...